python/eos/__init__.py

```diff
@@ -51,3 +51,15 @@ def FromWCxf(wc):
 
 from .data import *
 from .plot import *
+
+try:
+    if __IPYTHON__:
```

Comment:
Use similar `try ... except` to avoid `/usr/lib/python3/dist-packages/matplotlib/__init__.py:1352: UserWarning:  This call to matplotlib.use() has no effect
because the backend has already been chosen;
matplotlib.use() must be called *before* pylab, matplotlib.pyplot,
or matplotlib.backends is imported for the first time.`

that gets displayed in notebook

Reply:
That works nicely. I have disabled the selection of the PGF backend for any iPython based interpreter.